lib/foods.scss

```diff
@@ -1,3 +1,13 @@
 .foods-table {
   font-size: 14px;
 }
+
+#name-notice {
+  display: none;
+  color  : tomato;
+}
+
+#calories-notice {
+  display: none;
+  color: tomato;
+}
```

Comment:
Since these have the same properties, we can group them together.

` #name-notice, #calories-notice {
  display: none;
  color  : tomato;
}

https://www.thoughtco.com/grouping-multiple-css-selectors-3467065